src/core/Form/Select/MultiSelect/MultiSelect/MultiSelect.tsx

```diff
@@ -345,6 +359,34 @@ class BaseMultiSelect<T> extends Component<
     }
   };
 
+  private clickWasInToggleButton(event: MouseEvent | KeyboardEvent) {
+    return (
+      !!this.toggleButtonRef &&
+      (this.toggleButtonRef.current as Node).contains(event.target as Node)
+    );
+  }
+
+  private handleToggleButtonClick(event: React.MouseEvent<HTMLElement>) {
+    event.preventDefault();
+    if (!!this.filterInputRef && this.filterInputRef.current) {
+      if (document.activeElement !== this.filterInputRef.current) {
+        this.filterInputRef.current.focus();
+      } else {
+        this.setState((prevState: MultiSelectState<T & MultiSelectData>) => ({
+          showPopover: !prevState.showPopover,
+          showOptionsAvailableText: !prevState.showOptionsAvailableText,
+        }));
+      }
+    }
+  }
+
+  private disableItems(items: MultiSelectData[]): MultiSelectData[] {
+    return items.map((item) => ({
+      ...item,
+      disabled: true,
+    }));
+  }
```

Comment:
I think this could be cleaner and more uniform with other functions, if it was an arrow function, like this: 

```suggestion
   private disableItems = (items: MultiSelectData[]): MultiSelectData[] =>
    items.map((item) => ({
      ...item,
      disabled: true,
    }));
```

Reply:
Sure, changed it!